mongo/submission.py

```diff
@@ -720,27 +733,36 @@ def get_main_code(self) -> str:
         ext = self.main_code_ext
         return self.get_code(f'main{ext}')
 
-    def permission(self, user):
+    def own_permission(self, user):
         '''
-        3: can rejudge & grade, 
-        2: can view upload & comment, 
+        3: can rejudge & grade & give comment, 
+        2: can view upload & view feedback, 
         1: can view basic info, 
         0: can't view
         '''
```

Comment:
Maybe we can remove this doc string because

1. return type of `own_permission` is not `int` anymore.
2. the meaning of each enum value is documented in `Submission.Permission`

Reply:
https://github.com/Normal-OJ/Back-End/pull/188/commits/f37c48b2865dc8eea8a385a5f0f9e8a584ebde77